nilearn/datasets/func.py

```diff
@@ -15,7 +15,7 @@
 
 
 def fetch_haxby_simple(data_dir=None, url=None, resume=True, verbose=1):
-    """Download and load an example haxby dataset
+    """Download and load a simple example haxby dataset in a single list.
```

Comment:
in a single list is not important enough to feature in the short explanation (first line of the docstring). Also it is slightly inaccurate.